cmd/operator-sdk/alpha/scorecard/cmd.go

```diff
@@ -33,7 +34,15 @@ func NewCmd() *cobra.Command {
 		Long:   `Has flags to configure dsl, bundle, and selector.`,
 		Hidden: true,
 		Run: func(cmd *cobra.Command, args []string) {
-			log.Info("TODO")
+			scorecardFlags := scorecard.ScorecardFlags{
+				Config:   config,
+				Bundle:   bundle,
+				Selector: selector,
+				ListAll:  listAll,
```

Comment:
Just realized this field might be confusingly named since we will use the selector when listing, right? If so, maybe rename to `List`?

Reply:
changed in latest commit.